.github/workflows/merge.yml

```diff
@@ -15,8 +15,23 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Checkout init config repo
+        uses: actions/checkout@v3
+        with:
+          repository: SecureApiGateway/fr-platform-config
+          path: config-repo
+          token: ${{ secrets.MY_GITHUB_TOKEN }}
+          ref: master
```

Comment:
Just wondering how we will test changes that require the initializer code and config to be updated at the same time.

I don't want to have to merge the config change first, as it may break other environments if the config change is not backwards compatible.

Perhaps this ref could be looked up from a config file in this repo. Not sure if this is possible in a github action? We can then specify a sha commit hash or tag to pull the config from.

Reply:
I don't think that's possible, but we could instead use an env var in the pipeline for the ref, so you'd have to update the pipeline variable to test changes in the other repo, the PR will then run and you can see the outcome, but would have to remember to change the ref back to master before actually merging (this is the approach we had in my old place)

It's not ideal but it is a hangover from splitting out repos like this

Reply:
Wonder if we can move the git checkout into a bash script instead (and plug in the creds), and call it from the Makefile.

As it would be nice to be able to get the latest config whilst developing, otherwise locally you need to do a checkout of the repo manually and do the copy yourself.

Then the pipeline can call this make target and so can devs. 
It's not something I;ve done in github before but would assume it is possible.....

Reply:
So I think for the github action it should stay as it is, just to keep it simple instead of worrying about the bash script and credentials, but I've added steps to clone the config repo into the build step of the make file. It uses ssh to clone the repo which means if an ssh key isn't setup it won't work, but I'd imagine developers would all have this setup to be able to clone all our repos, assuming the make file is only for the benefit of us and not customers?

Reply:
See the note on the PR